components/eam/bld/config_files/definition.xml

```diff
@@ -348,12 +348,15 @@ Switch to enable or disable ECPP in CAM: 0=off, 1=on.
 <entry id="use_crm_cldfrac" valid_values="0,1" value="0">
 Switch to enable or disable fractional cloudiness in m2005 microphysics in CRM: 0=off, 1=on.
 </entry>
-<entry id="MMF_microphysics_scheme" valid_values="sam1mom,m2005" value="sam1mom">
+<entry id="MMF_microphysics_scheme" valid_values="sam1mom,m2005,p3" value="sam1mom">
 MMF CRM cloud microphysics scheme
 </entry>
-<entry id="crm" valid_values="sam,samomp,samxx" value="sam">
+<entry id="crm" valid_values="sam,samomp,samxx,pam" value="sam">
 MMF CRM model
 </entry>
+<entry id="pam_dycor" valid_values="awfl,spam" value="spam">
```

Comment:
Is there a comment or documentation somewhere that tells difference between awfl and spam ?

Reply:
No, but I should definitely add something somewhere. I don't really intend for this to be something that anyone would change, but I still think it will be useful to compare the two dycors even if we only really use one. We don't seem to have any good documentation of the dycor options in the PAM repo, so I'll add a comment in this file to clarify the difference.